string object_name
bool tiago
---
#result definition
# Comes from moveit_msgs/MoveItErrorCodes
int32 error_code
